package.json

```diff
@@ -109,5 +110,8 @@
     "last 2 versions",
     "IE 11"
   ],
-  "dependencies": {}
+  "dependencies": {},
+  "puppeteer": {
+    "chromium_revision": 768783
```

Comment:
Is someone going to manually keep this updated every 2 weeks or so?

Is it not possible to just keep puppeteer up to date? It is updated - https://github.com/puppeteer/puppeteer/releases, current is `782078`

Reply:
Hi @jbadan thanks for noticing it, updating version of puppeteer, without `chromium` will crash tests. I will try to find some way to keep chromium version up to date.

Reply:
@jbadan It's fixed, now it takes chrome version directly from puppeteer package

Reply:
Can you upgrade to latest puppeteer while we're already updating the tests - https://github.com/puppeteer/puppeteer/releases/tag/v5.2.1